### PR TITLE
XMPP (refinement)

### LINKED ISSUE #2272: XMPP icon
There were 2 changes in the XMPP logo, please look here:
- https://commons.wikimedia.org/wiki/File:XMPP_logo.svg

Old version, there was a problem: https://i.ibb.co/vcqmc8b/xmpp-logo-xsf-test.png

### PR DESCRIPTION
**Pull request by runxel**
**Issue:**
https://github.com/simple-icons/simple-icons/pull/2085#issuecomment-570833339
Closes #2272

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
I've taken the official version from Wikimedia (see comment above). Also updated the source.

### CODE REVIEW

**Comment by Neustradamus** [comment]
@runxel: Thanks a lot, if all people was like you!

File: _data/simple-icons.json [hunk 1]
```diff
@@ -4718,7 +4718,7 @@
         {
             "title": "XMPP",
             "hex": "002B5C",
-            "source": "https://xmpp.org/"
+            "source": "https://upload.wikimedia.org/wikipedia/commons/9/95/XMPP_logo.svg"
```
Comment (ericcornelissen) [inline _data/simple-icons.json]:
```suggestion
            "source": "https://commons.wikimedia.org/wiki/File:XMPP_logo.svg"
```

Comment (runxel) [reply _data/simple-icons.json]: Thanks! Updated it :)

**Comment by Neustradamus** [comment]
Linked to:
- https://github.com/simple-icons/simple-icons/pull/2085
- https://github.com/simple-icons/simple-icons/issues/2272
- https://github.com/simple-icons/simple-icons/pull/2275

**Pull request merged**